project/standard/static/karma.conf.single-run.js

```diff
@@ -15,14 +15,15 @@ module.exports = function karmaConfig(config) {
         }
     });
     testConfig.webpack.module.rules = [{
-                    test: /\.jsx?$/,
-                    exclude: /(__tests__|node_modules|legacy|libs\\Cesium|libs\\html2canvas)\\|(__tests__|node_modules|legacy|libs\/Cesium|libs\/html2canvas)\/|webpack\.js|utils\/(openlayers|leaflet)/,
-                    enforce: "pre",
-                    use: [
-                        {
-                            loader: 'babel-istanbul-loader'
-                        }
-                    ]
-                }, ...testConfig.webpack.module.rules];
+        test: /\.jsx?$/,
```

Comment:
Hi, i think this can be applied also to the [custom/static/conf](https://github.com/geosolutions-it/MapStore2/blob/master/project/custom/static/karma.conf.single-run.js#L51)

Reply:
custom project would need a complete review of the karma cfg, it's not that simple, so I skipped it for now, but if you have time you are welcome to fix it :-)